SPDX licenses in some cases cause a warning

<!--

   ************************************** WARNING **************************************

   The ciarcom bot parses this header automatically. Any deviation from the 
   template may cause the bot to automatically correct this header or may result in a 
   warning message, requesting updates.

   Please ensure all sections of the template below are filled in and no changes 
   are made to the template format. Only bugs should be raised here as issues. 
   Questions or enhancements should instead be raised on our forums:
   https://forums.mbed.com/ .

   *************************************************************************************

-->

### Description of defect

<!--
    Add detailed description of what you are reporting.
    Good example: https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html
-->

The recent update to the licenses causes compiler warnings

```
[351/731] Building C object CMakeFiles/mbed-os-example-soc.../source/Security/protocols/tls_sec_prot/tls_sec_prot.c.obj ../mbed-os/connectivity/nanostack/sal-stack-nanostack/source/Security/protocols/tls_sec_prot/tls_sec_prot.c: In function 'server_tls_sec_prot_state_machine':
../mbed-os/connectivity/nanostack/sal-stack-nanostack/source/Security/protocols/tls_sec_prot/tls_sec_prot.c:510:22: warning: unused variable 'remote_eui_64' [-Wunused-variable]
  510 |             uint8_t *remote_eui_64 = sec_prot_remote_eui_64_addr_get(prot);
      |                      ^~~~~~~~~~~~~
[472/731] Building CXX object CMakeFiles/mbed-os-example-s...ty/drivers/emac/TARGET_Freescale_EMAC/kinetis_emac.cpp.obj In file included from ../mbed-os/connectivity/netsocket/./include/netsocket/EMAC.h:23,
                 from ../mbed-os/connectivity/drivers/emac/TARGET_Freescale_EMAC/kinetis_emac.h:20,
                 from ../mbed-os/connectivity/drivers/emac/TARGET_Freescale_EMAC/kinetis_emac.cpp:48:
../mbed-os/connectivity/netsocket/./include/netsocket/EMACMemoryManager.h:2:1: warning: "/*" within comment [-Wcomment]
    2 | /* SPDX-License-Identifier: Apache-2.0
      |
```

This is  the license header

```
/* Copyright (c) 2017 ARM Limited
/* SPDX-License-Identifier: Apache-2.0
 *
```

I see there is `/` extra on the second line. I saw dozen of these during the build with CMake. 

cc @harmut01 
#### Target(s) affected by this defect ?

Any target


#### Toolchain(s) (name and version) displaying this defect ?

GCC ARM at least , prints a warning

#### What version of Mbed-os are you using (tag or sha) ?
<!--
    For a released version please provide the release tag (this can be found as per the instructions below)

    mbed-os version can be found in /platform/mbed_version.h. The tag can be reconstructed as follows:
    mbed-os-MBED_MAJOR_VERSION.MBED_MINOR_VERSION.MBED_PATCH_VERSION
 
    Master branch is indicated by 'mbed-os-99.99.99
    
    For an issue found on Master please provide the sha being used.
-->

8ef0a43


#### What version(s) of tools are you using. List all that apply (E.g. mbed-cli)

Not related

#### How is this defect reproduced ? 

Old tools should show the same warning (using same profiles as CMake).

--- a/connectivity/drivers/emac/TARGET_NUVOTON_EMAC/numaker_emac_config.h
+++ b/connectivity/drivers/emac/TARGET_NUVOTON_EMAC/numaker_emac_config.h
@@ -1,5 +1,5 @@
 /* Copyright (c) 2018 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/connectivity/drivers/emac/TARGET_NXP_EMAC/TARGET_LPCTarget/lpc17_emac.h
+++ b/connectivity/drivers/emac/TARGET_NXP_EMAC/TARGET_LPCTarget/lpc17_emac.h
@@ -1,5 +1,5 @@
 /* Copyright (c) 2018 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/connectivity/lwipstack/include/lwipstack/LWIPMemoryManager.h
+++ b/connectivity/lwipstack/include/lwipstack/LWIPMemoryManager.h
@@ -1,5 +1,5 @@
 /* Copyright (c) 2017 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/connectivity/lwipstack/source/LWIPInterface.cpp
+++ b/connectivity/lwipstack/source/LWIPInterface.cpp
@@ -1,5 +1,5 @@
 /* Copyright (c) 2017 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/connectivity/lwipstack/source/LWIPMemoryManager.cpp
+++ b/connectivity/lwipstack/source/LWIPMemoryManager.cpp
@@ -1,5 +1,5 @@
 /* Copyright (c) 2017 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/connectivity/nanostack/mbed-mesh-api/source/include/NanostackMemoryManager.h
+++ b/connectivity/nanostack/mbed-mesh-api/source/include/NanostackMemoryManager.h
@@ -1,5 +1,5 @@
 /* Copyright (c) 2017 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/connectivity/netsocket/include/netsocket/EMACMemoryManager.h
+++ b/connectivity/netsocket/include/netsocket/EMACMemoryManager.h
@@ -1,5 +1,5 @@
 /* Copyright (c) 2017 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/storage/filesystem/tests/TESTS/filesystem/general_filesystem/main.cpp
+++ b/storage/filesystem/tests/TESTS/filesystem/general_filesystem/main.cpp
@@ -1,5 +1,5 @@
 /* Copyright (c) 2017 ARM Limited
-/* SPDX-License-Identifier: Apache-2.0
+* SPDX-License-Identifier: Apache-2.0
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.

